packages/mercurius/package.json

```diff
@@ -24,6 +24,7 @@
     "url": "https://github.com/nestjs/graphql/issues"
   },
   "dependencies": {
+    "mercurius": "8.13.2",
```

Comment:
if i may, why this version ? why not 12 ?

Reply:
Tests fail if I increase the version I think some refactoring should be done in the tests, however I don't know if it will be a simple update there will still be a bit to do.

Reply:
@julestruong I was able to pass the tests by setting the version of mercurius to `v12.0.0` by making a change to an interface 🙌 ✅

Reply:
yeah i saw in the test logs it was the gateway impot, nice work.

Reply:
you only need to import `mercurius` on devDependencies though , right ?

Reply:
Can you please remove `mercurius` from dependencies? It's already flagged as a dev dependency (and a peer which is fine)

Reply:
i didn't notice thanks for the report i will update the PR

Reply:
Nothing fail the tests I should set in the devDependencies of la again the version "8.13.2":

Basically we should do this:

```
"devDependencies": {
    "@mercuriusjs/gateway": "^1.0.0",
    "@nestjs/common": "8.4.7",
    "@nestjs/platform-fastify": "8.4.7",
    "@nestjs/testing": "8.4.7",
    "fastify": "4.12.0",
    "mercurius": "8.13.2",
    "mercurius-integration-testing": "6.0.1"
  },
  "peerDependencies": {
    "@nestjs/common": "^8.2.3 || ^9.0.0",
    "@nestjs/graphql": "^10.0.0",
    "fastify": "^3.25.0 || ^4.0.0",
    "graphql": "^15.8.0 || ^16.0.0",
    "mercurius": "^8.12.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0"
  }
```

other ideas? 🤔

Reply:
Error seems to be 
`FastifyError: fastify-plugin: mercurius - expected '4.x' fastify version, '3.29.0' is installed`
how e2e loads node_modules ?